1.19_linux/runtime/sys_linux_arm64.s.patch

```diff
@@ -503,3 +503,49 @@ TEXT runtime·sysMapOS_trampoline(SB),NOSPLIT,$0
 	MOVD	0(R0), R0
 	BL	hitsumabushi_sysMapOS(SB)
 	RET
+
+TEXT runtime·fcntl_trampoline(SB),NOSPLIT,$0
+	MOVD	R0, R19		// R19 is callee-save
+	MOVW	8(R0), R1
+	MOVW	16(R0), R2
+	MOVW	0(R0), R0
+	BL	c_fcntl(SB)
+	MOVW	R0, 24(R19)	// return value
+	RET
+
+TEXT runtime·lseek_trampoline(SB),NOSPLIT,$0
+	MOVD	R0, R19		// R19 is callee-save
+	MOVD	8(R0), R1
+	MOVW	16(R0), R2
+	MOVW	0(R0), R0
+	BL	c_lseek(SB)
+	MOVW	R0, 24(R19)	// return value
+	RET
+
+TEXT runtime·fstat_trampoline(SB),NOSPLIT,$0
+	MOVD	R0, R19		// R19 is callee-save
+	MOVD	8(R0), R1
+	MOVW	0(R0), R0
+	BL	c_fstat(SB)
+	MOVW	R0, 16(R19)	// return value
+	RET
+
+TEXT runtime·renameat_trampoline(SB),NOSPLIT,$0
+	MOVD	R0, R19		// R19 is callee-save
+	MOVD	8(R0), R1
+	MOVD	16(R0), R2
+	MOVW	24(R0), R3
+	MOVD	0(R0), R0
+	BL	c_renameat(SB)
+	MOVW	R0, 32(R19)	// return value
+	RET
+
+TEXT runtime·fstatat_trampoline(SB),NOSPLIT,$0
```

Comment:
Let's order the functions in a consistent way with _amd64 version.

Reply:
Done!